### ISSUE TITLE
Hotkey combination is ignored

### ISSUE THREAD

**Issue by horst789github** [xanderfrangos/twinkle-tray#162]
i use the hotkey:
 "accelerator": "O+num8"
and
 "accelerator": "O+num2"

but the first button (O) is ignored and triggers already if you press only the 2nd button (num8 or num2)... in my example i just have to press the 2 or the 8 on the numberpad and the brightness changes, although i actually have that the hotkey may only change with a combination of O and 8 or O and 2...

![grafik](https://user-images.githubusercontent.com/35397597/113567936-d492d300-960f-11eb-977d-d8596a98e823.png)

{
  "isDev": false,
  "userClosedIntro": true,
  "theme": "default",
  "icon": "mdl2",
  "updateInterval": 500,
  "openAtLogin": true,
  "brightnessAtStartup": true,
  "killWhenIdle": false,
  "remaps": {
    "\\\\?\\DISPLAY#ACI23D2#5&119e9c2&0&UID4356#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}": {
      "min": 0,
      "max": 100
    },
    "\\\\?\\DISPLAY#ACI23D2#5&119e9c2&0&UID4353#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}": {
      "min": 0,
      "max": 100
    }
  },
  "hotkeys": {
    "\\\\?\\DISPLAY#GSM5B7F#5&119e9c2&0&UID4355#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}__dir1": {
      "monitor": "\\\\?\\DISPLAY#GSM5B7F#5&119e9c2&0&UID4355#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}",
      "accelerator": "O+num8",
      "direction": 1,
      "active": true
    },
    "\\\\?\\DISPLAY#GSM5B7F#5&119e9c2&0&UID4355#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}__dir-1": {
      "monitor": "\\\\?\\DISPLAY#GSM5B7F#5&119e9c2&0&UID4355#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}",
      "accelerator": "O+num2",
      "direction": -1,
      "active": true
    }
  },
  "hotkeyPercent": 15,
  "adjustmentTimes": [],
  "adjustmentTimeIndividualDisplays": false,
  "checkTimeAtStartup": true,
  "order": [
    {
      "id": "\\\\?\\DISPLAY#ACI23D2#5&119e9c2&0&UID4353#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}",
      "order": 0
    },
    {
      "id": "\\\\?\\DISPLAY#GSM5B7F#5&119e9c2&0&UID4355#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}",
      "order": 1
    }
  ],
  "features": {
    "luminance": true,
    "brightness": true,
    "gain": false,
    "contrast": false,
    "powerState": false,
    "volume": false
  },
  "checkForUpdates": true,
  "dismissedUpdate": "",
  "language": "system",
  "settingsVer": "v1.13.5",
  "names": {},
  "analytics": false,
  "scrollShortcut": true,
  "useAcrylic": false,
  "useNativeAnimation": false,
  "uuid": "c3860bb6-bf37-42fc-944e-07384c40622e",
  "branch": "master"
}

**Comment by xanderfrangos** [comment]
Yeah, I'm seeing it happen on my end when using the same shortcuts. Thanks for the bug report. I'll see if I can get this fixed.

**Comment by horst789github** [comment]
in the new v1.13.7 same problem

**Comment by horst789github** [comment]
in the new v1.13.9 same problem
